### PR TITLE
Fix Toshiba API connection

### LINKED ISSUE #72: Integration no longer compatible after Firmware Update to 4.x
there have been several issues on that topic: #68 #69 #71 - to easier manage the discussion and a possible solution I will close those issues in favor of this one.

**Toshiba announced a major change in their app and possibly also their backend**

screenshot taken from version 3.2 of <a href="https://play.google.com/store/apps/details?id=com.toshibatctc.SmartAC">toshiba home AC</a>

They claim that the end date of the "legacy" version (<=3.2) will no longer work after 14th of July. At least for my device (RAS-18PKVSG-E) this is not the case - I did not do the firmware nor the app update, and it keeps working - at least for now.

I have had a look at the new version of the app (see: https://github.com/h4de5/home-assistant-toshiba_ac/issues/68#issuecomment-1172667406) and currently I am not able to reverse engineer it. Unless I can find a way to do this, or there is an update of the underlying python package (https://github.com/KaSroka/Toshiba-AC-control) which is used to communicate with the server behind the AC unit, **we may be out of luck**.

This means if you have done the firmware update, you may no longer be able to control your AC unit via Home-Assistant. And I cannot do much about it. Some users claim, that a HA restart combined with a cold start of the AC unit, re-enables access, but this may depend on the AC unit itself.

If you are affected by that issue, you can simple hit the subscribe button on the right to stay informed. Please leave the comments only for possible solutions.

Best regards

### PR DESCRIPTION
**Pull request by tetienne**
Fix #72

### CODE REVIEW

File: custom_components/toshiba_ac/climate.py [hunk 1]
```diff
@@ -291,14 +302,20 @@ def target_temperature(self) -> float | None:
...
+            and self._device.ac_merit_a == ToshibaAcMeritA.HEATING_8C
+        ):
             return convert_temperature(5, TEMP_CELSIUS, self.temperature_unit)
         return convert_temperature(17, TEMP_CELSIUS, self.temperature_unit)
 
     @property
     def max_temp(self) -> float:
         """Return the maximum temperature."""
-        if hasattr(self._device, "ac_merit_a") and self._device.ac_merit_a == ToshibaAcMeritA.HEATING_8C:
+        if (
```
Comment (tetienne) [inline custom_components/toshiba_ac/climate.py:315]: @h4de5 Your last commit was not compliant with black. You can avoid this behavior if you create a PR for each modifications.

**Review by KaSroka — APPROVED** [review]
Looks good for me

**Comment by tetienne** [comment]
@h4de5 Can we merge and release this PR, it will unblock everybody?

File: custom_components/toshiba_ac/climate.py (hunk 1 above)
Comment (h4de5) [reply custom_components/toshiba_ac/climate.py:315]: I sincerly thought all files are already compliant with black. I just changed the manifest/requirements files

Comment (tetienne) [reply custom_components/toshiba_ac/climate.py:315]: Perhaps have you another formatter setup locally.

**Pull request merged**